test/specs/collections/Breadcrumb/BreadcrumbSection-test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import BreadcrumbSection from 'src/collections/Breadcrumb/BreadcrumbSection'
+import * as common from 'test/specs/commonTests'
+import sandbox from 'test/utils/Sandbox-util'
+
+describe('BreadcrumbSection', () => {
+  common.isConformant(BreadcrumbSection)
+  common.rendersChildren(BreadcrumbSection)
+  common.propKeyOnlyToClassName(BreadcrumbSection, 'active')
+
+  it('is div by default and does not have `href` attr', () => {
```

Comment:
Let's split this into two tests:

```js
it('renders as a div by default', ...)
```

And move the `href` assertion to the `describe('href')` section:

```js
describe('href prop', () => {
  it('is not present by default', () => {
    // ...
  })
})
```